assets/blocks/floatyMineral.block

```diff
@@ -0,0 +1,5 @@
+{
+    "displayName": "floatyMineral",
+    "categories" : ["floatyMineral"],

```

Comment:
The block categories are used for things like determining which tools are appropriate for mining a block. I don't think that a "floatyMineral" category makes much sense, and this should have the "mineral" or "rock" category (in fact, just giving it CoreAssets:mineral as a template would make sense). Also, the display name is presumably used for actual in-game UIs, so it shouldn't be camel case. It should probably be something a bit fancier sounding than just "Floaty mineral" though, like an actual fictional name rather than just something descriptive.